cmake.d/BundleQuiccir.cmake

```diff
@@ -0,0 +1,46 @@
+#
+# Get system Quiccir if available otherwise fetch it
+#
+message(DEBUG "BundleQuiccir")
+list(APPEND CMAKE_MESSAGE_INDENT "${QUICC_CMAKE_INDENT}")
+
+set(QUICC_QUICCIR_VERSION "v1.0.0-rc1")
```

Comment:
We should aim for v1.0.0 before merge

Reply:
yes, I will cleanup the `quiccir` side next week